content/en/catalyst/fund9/challenges/legal-and-finacial-implementation/campagin-brief.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "legal & finacial implementation"
+linkTitle: "legal & finacial implementation"
+type: docs
+hide_feedback: true
+description: "How can we build effective solutions to explore and provide legal & financial services and support to funded Proposers."
+---
+<!-- <img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/21-Community-Advisor-Improvements-dbdbe2.png" style="width:500px;height500px"> -->
+
+Fund Size: **500,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/399771) and [browse ideas]().
```

Comment:
Do we need to change the campaign link to this link, 'https://cardano.ideascale.com/c/campaigns/26596/about'?